pkg/util/util.go

```diff
@@ -48,6 +48,11 @@ var (
 	IsOpenShift, IsOpenShift4, _ = DetectOpenShift()
 )
 
+const (
+	DefaultServerExposureStrategyName = "multi-host"
+	DefaultSingleHostExposureStrategyName = "single-host"
+)
+
```

Comment:
```suggestion
```

Reply:
Actually we don't have default "multi-host" anymore since it depends on different use cases.
So, we can remove this default.